docker/drama-free-django/Dockerfile

```diff
@@ -0,0 +1,23 @@
+FROM centos:7
+
+ENV SCL_PYTHON_VERSION python27
+
+ENV DFD_DIR /src/cfgov-refresh
+
+# Must be world writable since alternate uid:gid may be patched in at `docker run` time.
+RUN mkdir -p ${DFD_DIR} && chmod 777 ${DFD_DIR}
+WORKDIR ${DFD_DIR}
+
+# Install dependencies
+RUN yum install -y centos-release-scl && \
+    curl -sL https://rpm.nodesource.com/setup_10.x | bash - && \
+    curl -sL https://dl.yarnpkg.com/rpm/yarn.repo | tee /etc/yum.repos.d/yarn.repo && \
+    yum install -y ${SCL_PYTHON_VERSION} gcc git nodejs which yarn && \
```

Comment:
Is the inclusion of `which` here leftover from debugging?

Reply:
Yes, I can remove that.  I find it strange that CentOS doesn't have `which` in its base distribution, but yeah.